packages/plugins/deploy-tracker/src/trackingFunctions.js

```diff
@@ -111,6 +111,44 @@ export default class TrackingFunctions {
     cb();
   }
 
+  async trackMigrationScript(params, callback) {
+    if (!this.enabled) {
+      return callback();
+    }
+
+    const {
+      scriptName,
+      scriptDirectory,
+      forceTracking,
+      migrationsDirectory
+    } = params;
+
+    if (!forceTracking && scriptDirectory !== migrationsDirectory) {
+      return callback();
```

Comment:
Not gonna comment on each of them, but the `params` should be passed to every callback.